demo-shell-ng2/app/app.component.ts

```diff
@@ -44,11 +44,17 @@ export class AppComponent {
 
         if (translateService) {
             if (process.env.ENV === 'production') {
+                // This adds i18n strings that are specific to demo-shell app
                 translateService.addTranslationFolder('custom', 'i18n/custom-translation');
-                translateService.addTranslationFolder('ng2-alfresco-login', 'i18n/custom-translation/alfresco-login');
+                // Uncomment this to override component i18n
```

Comment:
I see a lot of comments here....do we need more documentation about translation....maybe in the core?can we update the README.md?